How-To-Release-Riak.md

```diff
@@ -0,0 +1,56 @@
+* A Go/No-Go scheduled on the Thursday after the expected delivery date or fallback date to ensure that the public release can proceed as planned. During the Go/No-Go we must ensure:
+  * Giddyup has been triaged, and there are no major issues
+  * Smoke testing has completed successfully on all supported platforms
+  * Documentation content has been merged and triaged if necessary, and release notes are ready to ship
+  * Packages for all supported platforms have been built and are ready to stage
+  * Any marketing announcements have been reviewed and are ready to ship
+* A final staging date on the Friday after the Go/No-Go. During this time, all packages will be pushed to their final download locations, the `External Release` docs will be sent around internall for review, and the indexes for the download pages will be generated
+* A public release date on the Tuesday after the final staging date. This date cannot overlap with another public release.
+
+# Pre-Release
+All releases of Riak will have at least one `Release Candidate` before the final package build. These RCs will contain all of the `code, backports, and bugfixes` required to build the Final Release, but will not have any updated `Release Notes`. Release Notes updates will be the only difference between RCs and the final version that will be released to the public.
+
+## Pre-release Procedure
+All Release Candidates will undergo the following release stages to ensure quality:
+  * Package Build
+  * Internal Publishing
+  * riak_test (giddyup) runs
+  * Smoke Testing (on at least the final RC)
+
+The following procedure will cover all of these steps:  
+* Ensure `riak_test` is configured to automatically run after the package building steps:
+  * If there is no branch in `riak_test` that corresponds to `riak`, create it
+  * `basho_builds.yml` must exist in the top level of the repo, and contain the list of devrels that will be used to test various upgrade / downgrade scenarios during riak_test runs. The contents of the file will be similar to:
+ ```
+docker_rt_riak_test_config:
+  - version: current
+    product_version: current
+    product: riak_ee
+  - version: previous
+    product_version: 2.2.0
+    product: riak_ee
+  - version: legacy
+    product_version: 2.0.8
+    product: riak_ee
+  - version: 2.0.2
+    product_version: 2.0.2
+    product: riak_ee
+  - version: 2.0.4
+    product_version: 2.0.4
+    product: riak_ee
+  - version: 2.0.5
+    product_version: 2.0.5
+    product: riak_ee
+```
+  * Tag riak_test with the same `annotated tag` that you will use for riak
+  * Push the tag to github  
+* Bundle Riak
+  * Lock the deps: `make lock`
+  * Tag riak with an `annotated tag`  
+* Giddyup runs will be kicked off automatically after the package builds complete successfully, but results must be triaged after each run. Giddyup runs usually complete 4.5 hours after the tag is initially received by the system.  
```

Comment:
Is there a document on triaging Jenkins and or Ansible Tower? Not sure this is the right place for it, but a pointer might be nice since it's part of the release cycle.